app/views/account/_available_items.html.erb

```diff
@@ -1,31 +1,29 @@
 <h2 class="section-heading">Items Available for Pickup</h2>
 <% if (!@account.avail_items.nil?) %>
-  <div class="table-responsive">
-    <table class="table table-striped table-bordered account--available_items">
-      <thead>
+  <table class="table table-striped table-bordered account--available_items">
+    <thead>
+      <tr>
+        <th>Cancel</th>
+        <th>Item Title</th>
+        <th>Request Expiration Date</th>
+        <th>Pickup Location</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @account.avail_items.each do |item| %>
         <tr>
-          <th>Cancel</th>
-          <th>Item Title</th>
-          <th>Request Expiration Date</th>
-          <th>Pickup Location</th>
+         <td>
+            <label><input id="cancel-<%= item['itemID'] %>" type="checkbox" name="cancel_requests[]"
+              value="<%= format_hold_cancel(item) %>">
+              <span class="sr-only">Cancel</span></label>
+          </td>
+          <td><%= link_to(item["itemTitle"], catalog_path(item["itemId"])) %></td>
```

Comment:
In this case the itemID is literally the Voyager item ID. It isn't the bib ID so the catalog_path helper isn't going to produce a good link.